### PR TITLE
New hacktober 2023 - Normalize String 

### PR DESCRIPTION
**Pull request by SiddhantTotade**
### Describe your change:
A program which normalizes the given string.

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: strings/normalize_string.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    "any",
+    "it",
+    "after",
+    "down",
+    "into",
+    "we",
+    "no",
+    "other",
+    "who",
+    "wasn",
```
Comment (haxkd) [inline strings/normalize_string.py:65]: was

File: strings/normalize_string.py [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    "wasn't",
+    "own",
+    "did",
+    "couldn",
```
Comment (haxkd) [inline strings/normalize_string.py:69]: could

File: strings/normalize_string.py [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    "if",
+    "weren't",
+    "has",
+]
+
+
+# Function for normalizing the string
+def normalize_string(text: str) -> list:
+    """
+    This fuction normalizes the given string
```
Comment (haxkd) [inline strings/normalize_string.py:197]: This function normalizes the given string

File: strings/normalize_string.py [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+    # splitting the string into words
+    words = remove_whitespace.split()
+
+    # filtering words and removing the stop words
+    filtered_words = [word for word in words if word.lower() not in stop_words]
+    remove_stop_words = " ".join(filtered_words)
+
+    normalized_text = remove_stop_words.split()
+
+    # returing the list
```
Comment (haxkd) [inline strings/normalize_string.py:237]: returning the list

**Review by haxkd — CHANGES_REQUESTED** [review]

**Comment by cclauss** [comment]
Not interested.

**Pull request closed without merging**